<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qtpower</class>
 <widget class="QWidget" name="Qtpower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qtpower</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>对话框弹出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_state">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_init_timer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化时钟</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_stop_timer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止时钟</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
